src/controller/tests/users.controller.test.ts

```diff
@@ -0,0 +1,196 @@
+import controller from '../users.controller'
+
+import User from '../../model/users.model'
+
+import UserService from '../../services/user.service'
+
+import Testing from '../../utils/testing.utils'
+import { GenericResponse, NotFound, Updated } from '../../utils/apiResponse.utils'
+
+// Testing Globals
+let req
+let res
+let next
+
+let expectedResults
+let expectedResult
+let expectedError
+
+describe('UserController', () => {
+  beforeEach(() => {
+    req = Testing.fakeRequest()
+    res = Testing.fakeResponse()
+    next = Testing.fakeNext()
+
+    expectedResults = Testing.generateTypeOrmArray<User>(User, 3)
```

Comment:
@daviddob alrighty, I didn't add faker (for now) and instead gave us a way to generate a few models easily for testing. 

If we find we need better fake data creation (random names, etc), we can do so later.